list-your-project/other-dashboards/README.md

```diff
@@ -58,60 +59,122 @@ Let's start with a simple, complete example of a fees adapter:
 
 ### Adapter Structure
 
-The object exported by your adapter file defines its behavior. The main configuration object holds a `version` key and an `adapter` key.
+The object exported by your adapter file defines its behavior. The main configuration object holds a `version` key and supports two different structures:
+
+**Recommended Structure (Root-level fetch)**: For protocols with the same fetch logic across all chains, you can use the simplified structure with `fetch`, `chains`, `start`, and `methodology` at the root level. This reduces code repetition.
+
+**Legacy Structure (Chain-specific configurations)**: For protocols that require custom fetch logic, start dates, or methodology per chain, you can still use the `adapter` object with chain-specific configurations, keyed by `CHAIN.<ChainName>`. Each chain configuration is a `BaseAdapter` object.
 
-The `adapter` object contains chain-specific configurations, keyed by `CHAIN.<ChainName>`. Each chain configuration is a `BaseAdapter` object.
+### SimpleAdapter Properties
 
-### BaseAdapter Properties
+#### Root-level Properties (Recommended)
 
-The `BaseAdapter` includes several important properties:
+For protocols with consistent logic across chains, use these root-level properties:
 
 * **fetch**: The core async function that returns different dimensions of a protocol. The dimensions returned depend on which dashboard you're targeting (e.g., `dailyVolume` for the dexs dashboard, `dailyFees` for the fees dashboard). See "Core Dimensions" below.
+* **chains**: Array of chain constants (e.g., `[CHAIN.ETHEREUM, CHAIN.POLYGON]`) indicating which chains this adapter supports.
 * **start**: The earliest timestamp (as YYYY-MM-DD or unix timestamp) we can pass to the fetch function. This tells our servers how far back we can get historical data.
+* **methodology**: (Optional) Object describing how different dimensions are calculated. See "Metadata and Methodology" below.
 * **runAtCurrTime**: (Optional, defaults to `false`) Boolean flag. Set to `true` if the adapter can only return the latest data (e.g., last 24h) and cannot reliably use the `startTimestamp` and `endTimestamp` passed to `fetch`.
+
+#### Legacy Chain-specific Properties (BaseAdapter)
+
+For protocols requiring custom configuration per chain, use the `adapter` object with `BaseAdapter` configurations:
+
+* **fetch**: The core async function that returns different dimensions of a protocol.
+* **start**: The earliest timestamp (as YYYY-MM-DD or unix timestamp) we can pass to the fetch function.
+* **runAtCurrTime**: (Optional, defaults to `false`) Boolean flag.
 * **meta**: (Optional) Object containing metadata about the adapter, including:
-  * **methodology**: Object describing how different dimensions are calculated. See "Metadata and Methodology" below.
+  * **methodology**: Object describing how different dimensions are calculated.
   * **hallmarks**: Set of events that significantly affected protocol data (displayed on the chart).
```

Comment:
I have moved all hallmarks from dimensions adapters to data.ts, so please remove this line